resources/wallet-generator/monero-wallet-generator.html

```diff
@@ -11063,195 +8366,94 @@ <h4 style="color: #555;">Offline wallet</h4>
+                            <input type="button" class="waves-effect waves-light btn btn-qr" onclick="js:toggle_priv_mnemonic_qr();" value="Show QR code" action="" id="show_priv_mnemonic_qr_code"/>
+                            </span>
+                            <p class="noprint">
+                                The mnemonic seed is a string that comprises 25 words and allows you to recreate and backup your wallet. <b>Keep it secure and don't share it.</b>
+                            </p>
+                            <p class="print-only" id="mnemonic_address">Generating...</p>
                         </div>
-                        <div class="col s9">
-                            <span id="spend_key_widget">Generating...</span>
-                            <span class="qrcode" id="priv_spend_qr_widget" onclick="js:toggle_priv_spend_qr();"></span>
+                        <div class="card-action center-align">
+                            <h5 class="grey-text text-darken-4" id="mnemonic_widget">Generating... </h5>
+                            <span class="qrcode" id="priv_mnemonic_qr_widget" onclick="js:toggle_priv_mnemonic_qr();"></span>
                         </div>
                     </div>
-                    <div class="row">
-                        <div class="col s3">
-                            <span class="label">View key:</span>
-                        </div>
-                        <div class="col s9">
-                            <span id="view_key_widget">Generating...</span>
-                            <span class="qrcode" id="priv_view_qr_widget" onclick="js:toggle_priv_view_qr();"></span>
-                        </div>
-                    </div>
-                    <div class="row">
-                        <div class="col s6">
-                            <input type="button" class="waves-effect waves-light btn btn-qr" onclick="js:toggle_priv_spend_qr();" value="Show Spend Key as QR code" action="" id="show_priv_spend_qr_code"/>
+            </div>
+            <div class="row-new print_break_before">
+                    <div class="card pink lighten-5">
+                        <div class="card-content pink-text text-darken-4">
+                            <span class="card-title pink-text text-darken-4">Private keys <span class="noprint">(optional)</span></span>
+
+                            <p class="noprint">
+                                The spend key and view key are the raw private keys for the
+                                new wallet.
+                                <b>They are here for your information, since they can be recovered using the mnemonic
+                                seed in the above box</b>.
+                                If you decide to keep them, keep them secure and don't share them.
+                            </p>
+                            <p class="print-only" id="privatekeys_address">Generating...</p>
```

Comment:
This used to be blue, not pink. Was the choice to make it the same color as the seed intentional? (Public info = green, private info = pink). If so, seems good.

Reply:
Yes, the original used three different colours. I thought of simplifying by making visually clear what is safe to share publicly (green) and what is meant to keep secret (pink). Thanks for the review, i'll add your other suggestions tomorrow.